CMakeLists.txt

```diff
@@ -574,6 +574,27 @@ add_library (seastar STATIC
   src/http/transformers.cc
   src/json/formatter.cc
   src/json/json_elements.cc
+  src/kafka/protocol/kafka_error_code.hh
```

Comment:
@psarna this is not in any IF so I assume this is wrong and won't be guarded by a flag, right?

Reply:
Right, forgot about the conditionals, I'll fix it in a sec

Reply:
Yes, and first and foremost its target branch is `psarna/master`, which is a random branch from a long while ago, which does not contain this commit, which introduces conditional compilation: ae2232f41871d907838befa6228c22a9fe6b7d5a . It should probably be built on top of `psarna/zpp_kafka`.

Reply:
Ok, I'll keep it here for now to keep the review continuous and once done I'll incorporate the guards and make a PR with `parna/zpp_kafka` as base